src/pocketmine/Player.php

```diff
@@ -834,12 +839,10 @@ public function getLocale() : string{
 	 * Plugin developers should not use this, use setSkin() and sendSkin() instead.
 	 *
 	 * @param Skin   $skin
-	 * @param string $newSkinName
-	 * @param string $oldSkinName
 	 *
 	 * @return bool
 	 */
-	public function changeSkin(Skin $skin, string $newSkinName, string $oldSkinName) : bool{
+	public function changeSkin(Skin $skin) : bool{
```

Comment:
These are here for future usage. If their removal isn't related to a protocol change, leave them alone.